dapr-spring/dapr-spring-boot-autoconfigure/src/main/java/io/dapr/spring/boot/autoconfigure/client/DaprClientAutoConfiguration.java

```diff
@@ -58,4 +63,29 @@ DaprClient daprClient(DaprClientBuilder daprClientBuilder) {
     return daprClientBuilder.build();
   }
 
+  @Bean
+  @ConditionalOnMissingBean
+  DaprWorkflowClient daprWorkflowClient(DaprConnectionDetails daprConnectionDetails) {
+    final Properties properties = createPropertiesFromConnectionDetails(daprConnectionDetails);
+    return new DaprWorkflowClient(properties);
+  }
+
+  @Bean
+  @ConditionalOnMissingBean
+  WorkflowRuntimeBuilder daprWorkflowRuntimeBuilder(DaprConnectionDetails daprConnectionDetails) {
+    final Properties properties = createPropertiesFromConnectionDetails(daprConnectionDetails);
+    return new WorkflowRuntimeBuilder(properties);
+  }
+
+  private Properties createPropertiesFromConnectionDetails(DaprConnectionDetails daprConnectionDetails) {
```

Comment:
We don't need `final` here it just adds syntax noise.

Reply:
done